lnrpc/invoicesrpc/invoices.proto

```diff
@@ -99,3 +99,8 @@ message SettleInvoiceMsg {
 } 
 
 message SettleInvoiceResp {}
+
+message SubscribeSingleInvoiceRequest {
+    /// Hash corresponding to the (hold) invoice to subscribe to.
+    bytes r_hash = 2 [json_name = "r_hash"];
```

Comment:
should set `reserved  1` to indicate field 1 should not be used.

Reply:
Thanks, done.